<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageGeneral</class>
 <widget class="QWidget" name="PageGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="hlayOpening">
     <item>
      <widget class="QGroupBox" name="boxOpening">
       <property name="title">
        <string>Opening</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="chboxSingleInstance">
          <property name="text">
           <string>Open files in existing instance</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayLanguage">
     <item>
      <widget class="QGroupBox" name="boxLanguage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Language</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cmbLanguage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hspacerLanguage">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="boxLogging">
     <property name="title">
      <string>Logging</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="hlayLogLevel">
        <item>
         <widget class="QLabel" name="lblLogLevel">
          <property name="text">
           <string>Logging level</string>
          </property>
          <property name="buddy">
           <cstring>cmbLogLevel</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbLogLevel"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chboxConsoleLog">
        <property name="text">
         <string>Console logging</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chboxFileLog">
        <property name="text">
         <string>File logging</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlayLogFilePath">
        <item>
         <widget class="QLabel" name="lblLogFilePath">
          <property name="text">
           <string>Log file</string>
          </property>
          <property name="buddy">
           <cstring>editLogFilePath</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editLogFilePath"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnLogFilePathDlg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>322</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
